src/createAnimatedComponent.js

```diff
@@ -229,7 +229,9 @@ export default function createAnimatedComponent(Component) {
     }
 
     _updateFromNative(props) {
-      this._component.setNativeProps(props);
+      if (this._component.setNativeProps) {
+        this._component.setNativeProps(props);
+      }
```

Comment:
`this._component.setNativeProps?.(props)`

Reply:
done a44f3d0